src/components/Login/Login.tsx

```diff
@@ -0,0 +1,110 @@
+import { FormEvent, useEffect, useState } from 'react';
+
+import { Button } from 'src/common/Button/Button';
+import Input from 'src/common/Input/Input';
+
+import { Link, useNavigate } from 'react-router-dom';
+
+import * as text from 'src/constants';
+
+const Login = () => {
+	const url = 'http://localhost:4000/login';
+
+	const forbiddenSymbols = /[@#$%^&]/;
```

Comment:
Just to keep it simple, you can add `type="email"` to the Input to delegate validation of the input to the browsers. I can't type "@" now